_rules/form-control-non-empty-accessible-name-e086e5.md

```diff
@@ -97,61 +103,54 @@ _There are currently no assumptions_
 
 #### Passed Example 5
 
-[Explicit role](#explicit-role).
+This element with a `combobox` [role][semantic role] has an [accessible name][] because of its `aria-label` attribute.
 
 ```html
+<div>Country</div>
 <div aria-label="country" role="combobox" aria-disabled="true">England</div>
 ```
 
-#### Passed Example 6
-
-The [accessible name][] is not only [whitespace][].
-
-```html
-<label>
-	:-)
-	<input />
-</label>
-```
-
 ### Failed
 
 #### Failed Example 1
 
-No [accessible name][].
+This `input` element has no attribute to give it an [accessible name][].
 
 ```html
+<div>last name</div>
 <input />
 ```
 
 #### Failed Example 2
 
-[Non-focusable](#focusable) element still needs an [accessible name][].
+This [non-focusable](#focusable) `input` element has no attribute to give it an [accessible name][].
```

Comment:
"non-fousable" is not really correct. It's out of sequential focus navigation, but still programmatically focusable.

Reply:
I'm replacing this with a disabled element. Makes for a better example IMO.